### PR TITLE
Adding prototype for registration page.

### PR DESCRIPTION
**Pull request by KevinWorkman**
Added basic setup for registration page. Note that this does not actually handle user input yet.

### CODE REVIEW

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+public class RegisterServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException, ServletException {
+    request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
+  }
+}
```
Comment (wgenemorgan) [inline src/main/java/codeu/controller/RegisterServlet.java]: Add blank line at end?

File: src/main/webapp/WEB-INF/view/register.jsp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <input type="text" name="username" id="username">
+      <br/>
+      <label for="password">Password: </label>
+      <input type="password" name="password" id="password">
+      <br/>
+      <button type="submit">Submit</button>
+    </form>
+  </div>
+</body>
+</html>
```
Comment (wgenemorgan) [inline src/main/webapp/WEB-INF/view/register.jsp]: Add blank line at end.

File: src/main/webapp/WEB-INF/view/register.jsp [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
```
Comment (wgenemorgan) [inline src/main/webapp/WEB-INF/view/register.jsp]: Add space after "else"

File: src/main/webapp/WEB-INF/view/register.jsp [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+<head>
+  <title>Register</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
```
Comment (wgenemorgan) [inline src/main/webapp/WEB-INF/view/register.jsp]: Add space before {

File: src/test/java/codeu/controller/RegisterServletTest.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        .thenReturn(mockRequestDispatcher);
+  }
+
+  @Test
+  public void testDoGet() throws IOException, ServletException {
+    registerServlet.doGet(mockRequest, mockResponse);
+
+    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+  }
+}
```
Comment (wgenemorgan) [inline src/test/java/codeu/controller/RegisterServletTest.java]: Add blank line at end.

File: src/main/java/codeu/controller/RegisterServlet.java (hunk 1 above)
Comment (KevinWorkman) [reply src/main/java/codeu/controller/RegisterServlet.java]: Done.

File: src/main/webapp/WEB-INF/view/register.jsp (hunk 2 above)
Comment (KevinWorkman) [reply src/main/webapp/WEB-INF/view/register.jsp]: Done.

File: src/main/webapp/WEB-INF/view/register.jsp (hunk 3 above)
Comment (KevinWorkman) [reply src/main/webapp/WEB-INF/view/register.jsp]: Done.

File: src/main/webapp/WEB-INF/view/register.jsp (hunk 4 above)
Comment (KevinWorkman) [reply src/main/webapp/WEB-INF/view/register.jsp]: Done.

File: src/test/java/codeu/controller/RegisterServletTest.java (hunk 5 above)
Comment (KevinWorkman) [reply src/test/java/codeu/controller/RegisterServletTest.java]: Done.

**Comment by KevinWorkman** [comment]
Thanks for the review! Merging now.

**Pull request merged**